### ISSUE TITLE
Cannot Use with Server Side Rendering

### ISSUE THREAD

**Issue by stripathi669** [stefalda/react-localization#53]
navigator is not present in server side. Hence, the function `getInterfaceLanguage` throws Reference error.

Any workaround?

**Comment by stefalda** [comment]
Why don't you define a global navigator object (a sort of polyfill) with the called method?

**Comment by jonanderdev** [comment]
A simple check could be added, to see if it's running on a server to not make use of that navigator object. 

This is just an idea. What do you guys think?
```
export function getInterfaceLanguage() {
  // Check if is running on server side
  if (typeof window === 'undefined') { 
     return 'en-US';
  }

  if (!!navigator && !!navigator.language) {
    return navigator.language;
  } else if (!!navigator && !!navigator.languages && !!navigator.languages[0]) {
    return navigator.languages[0];
  } else if (!!navigator && !!navigator.userLanguage) {
    return navigator.userLanguage;
  } else if (!!navigator && !!navigator.browserLanguage) {
    return navigator.browserLanguage;
  }
  return 'en-US';
}
```

**Comment by N4G1B4T0R** [comment]
Hi! I got the same issue with SSR but i am using a lib next.js for universal app.

Here is my package.json.

`"dependencies": {
    "next": "latest",
    "node-sass": "^4.7.2",
    "react": "^16.2.0",
    "react-dom": "^16.2.0",
    "react-localization": "^0.1.8",
    },`
So, I got this error "ReferenceError: navigator is not defined" message. 

To fix this issue in function getInterfaceLanguage() I just added global object. 

`if (!!global.navigator && !!global.navigator.language) {
    return global.navigator.language;
  } else if (!!global.navigator && !!global.navigator.languages && !!global.navigator.languages[0]) {
    return global.navigator.languages[0];
  } else if (!!global.navigator && !!global.navigator.userLanguage) {
    return global.navigator.userLanguage;
  } else if (!!global.navigator && !!global.navigator.browserLanguage) {
    return global.navigator.browserLanguage;
  }`

The reason is that on server we haven't got  object window.

I will make a commit to fix this issue. If u can plz check ASAP.
